analyzers/its/expected/Net5/Net5--net5.0-S2952.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
I did not review the removed ITs, since you mentioned that duplicates exists in the unit tests.

Reply:
Please do so anyhow. I linked the source files in the comments on GitHub.
https://github.com/SonarSource/sonar-dotnet/pull/6230#discussion_r1108447527